package.json

```diff
@@ -40,31 +40,40 @@
     "prepare_release:experimental:patch": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:remove_stable && npm run _lerna:version_patch && npm run _restore:package-json && npm run _changelog:prepare_experimental",
     "prepare_release:experimental:minor": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:remove_stable && npm run _lerna:version_minor && npm run _restore:package-json && npm run _changelog:prepare_experimental",
     "comment_prepare_3": "echo sdk preparation scripts prepare all stable and experimental packages",
-    "prepare_release:sdk:patch": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:version_patch && npm run _restore:package-json && npm run _changelog:prepare_experimental && npm run _changelog:prepare_stable",
-    "prepare_release:sdk:minor": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:version_minor && npm run _restore:package-json && npm run _changelog:prepare_experimental && npm run _changelog:prepare_stable",
+    "prepare_release:sdk:patch": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:remove_semconv && npm run _lerna:version_patch && npm run _restore:package-json && npm run _changelog:prepare_experimental && npm run _changelog:prepare_stable",
+    "prepare_release:sdk:minor": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:remove_semconv && npm run _lerna:version_minor && npm run _restore:package-json && npm run _changelog:prepare_experimental && npm run _changelog:prepare_stable",
+    "comment_prepare_4": "echo semconv preparation scripts only prepare semconv package",
+    "prepare_release:semconv:patch": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:remove_stable_except_semconv && npm run _lerna:remove_experimental && npm run _lerna:version_patch && npm run _restore:package-json && npm run _changelog:prepare_semconv",
+    "prepare_release:semconv:minor": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:remove_stable_except_semconv && npm run _lerna:remove_experimental && npm run _lerna:version_minor && npm run _restore:package-json && npm run _changelog:prepare_semconv",
```

Comment:
Hmm, both of these will also bump examples and such, so they will diverge over time from the experimental/stable versioning over time. Maybe we should also exclude them from the update. :thinking:

Reply:
You mean we want to ensure absolutely nothing is included in this EXCEPT for the semantic-conventions package, right? Can you look here if this matches what you're suggesting: https://github.com/open-telemetry/opentelemetry-js/pull/4904/commits/1d7a1bd69b7e4d7e98731408e1487c07fa1062f3